test/coverlet.core.tests/Reporters/CoberturaReporterTests.cs

```diff
@@ -146,43 +146,65 @@ public void TestReportWithTwoDifferentDirectories()
 
             string absolutePath1;
             string absolutePath2;
+            string absolutePath3;
+            string absolutePath4;
+            string absolutePath5;
+            string absolutePath6;
 
             if (isWindows)
```

Comment:
`isWindows` is used only in one place use directly `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)`

Reply:
Done